components/rhs_header_post/rhs_header_post.jsx

```diff
@@ -125,7 +124,6 @@ export default class RhsHeaderPost extends React.Component {
                     >
                         <i
                             className='fa fa-angle-left'
-                            title={localizeMessage('generic_icons.back', 'Back Icon')}
```

Comment:
We might want to re-add these using `aria-label` instead of `title`. Good catch that the title is redundant when we have nicer tooltips here, but I think we still need an `aria-label` since it's used by accessibility tools.

Reply:
@hmhealey updated.